spark-jobs/src/main/scala/za/co/absa/enceladus/common/plugin/menas/MenasRunUrl.scala

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2018 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.common.plugin.menas
```

Comment:
Why this file is in `plugins` package?

Reply:
Well, it is shared between Standardization & Conformance, so I figured that it should be somewhere in `spark-jobs`'s `commons`. And since it is menas-related, I have put that in the plugins/menas. Where would you suggest such a class should go? Just under commons?

Reply:
Agree. This is a good place too.